src/app/home/components/MovieCarousel/movie/DdayCarousel.tsx

```diff
@@ -1,11 +1,26 @@
 import * as styles from '@/app/home/components/MovieCarousel/MovieCarousel.css'
 
 import MovieCarousel from '../MovieCarousel'
+import { getdDay } from '@/app/services/getDday'
+import { DDayCard } from '@/shared/widget/MovieCard/MovieCard.types'
+
+const DdayCarousel = async () => {
+  const result = await getdDay()
+
+  const data: DDayCard[] = result.map((item, index) => ({
+    id: index,
+    untilRelease: item.untilRelease,
+    isWishedCount: item.isWishedCount,
+    imagePath: item.imagePath,
+    title: item.title,
+    releaseYear: item.releaseYear,
+    type: 'dDay',
+  }))
```

Comment:
```suggestion
const data: DDayCard[] = result.map(
  ({ untilRelease, isWishedCount, imagePath, title, releaseYear }, index) => ({
    id: index,
    untilRelease,
    isWishedCount,
    imagePath,
    title,
    releaseYear,
    type: 'dDay',
  }),
)
```
조금 더 간단해질 것 같아서 ........ !!